.gitignore

```diff
@@ -7,3 +7,5 @@ target/
 bazel-*
 .bazel-*
 .dazel_run
+
+scalafmt
```

Comment:
I've changed the script to install scalafmt locally so that it does not require sudo privs. This stops you accidentally adding a 17Mb jar to the project :-)